### PR TITLE
Movie tile

### PR DESCRIPTION
**Pull request by waltad**
Proszę o sprawdzenie przed merg-em kafelka filmu. Jak będzie ok to proszę o Piotrek dodaj do mastera lub daj znać, że ja już mogę. Pozdrawiam i życzę miłej zabawy. Będę wdzięczny za feedback.

### CODE REVIEW

**Comment by P-Michalski** [comment]
Hey! I checked everything and I have 2 minor things to be changed:
- number of votes and "votes" word should be grey instead of black,
- you should use justify-content: space-between only to catch Description and Rating, to create empty space between those 2 elements (simply you can rearrange something in your code to exclude Poster from beeing caught by that justify-content)
Apart from that, good job Tadeusz! 😄

File: src/common/MovieTile/styled.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  padding: 8px 16px 8px 16px;
+  border-radius: 5px;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 20px;
+  text-align: left;
+  background-color: #E4E6F0;
+`;
+
+export const Rating = styled.div`
```
Comment (P-Michalski) [inline src/common/MovieTile/styled.js:56]: you can add a color there, to match the one from figma (#7E839A in hex)

File: src/common/MovieTile/styled.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+export const Wrapper = styled.div`
+  width: 324px;
+  height: 650px;
+  border-radius: 5px;
+  box-shadow: 0px 4px 12px 0px rgba(186, 199, 213, 0.5);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
```
Comment (P-Michalski) [inline src/common/MovieTile/styled.js:11]: I think you could make a separate div to wrap only Description and Rating and use justify-content: space-between there instead

**Review by P-Michalski — CHANGES_REQUESTED** [review]
2 small changes and we can merge ^-^
Good job on that one!

File: src/common/MovieTile/styled.js (hunk 1 above)
Comment (waltad) [reply src/common/MovieTile/styled.js:56]: This color (#7E839A in hex) is too dark in my display. I stayed with the current color.

File: src/common/MovieTile/styled.js (hunk 2 above)
Comment (waltad) [reply src/common/MovieTile/styled.js:11]: I added 2 div-s and that did the trick

**Pull request merged**